CONTRIBUTING.md

```diff
@@ -9,6 +9,14 @@ Contributions are more than welcome. If you want to contribute, please make sure
 3. Open a Pull Request pointing to the `master` branch.
 4. A maintainer will review and, eventually, merge your Pull Request. Please, be patient as most of us are doing this in our spare time.
 
+### Develop
+
+1. Write code and tests.
+1. Make sure all tests pass `npm test`
+1. Make sure code is well formatted and secure `npm run lint`
+
+Release regenerates API documentation and browser bundle, so you do not have to regenerate it manually with `npm run docs` and `npm run prepublishOnly`.
+
```

Comment:
Somehow unrelated to this PR but shouldn't we use the CONTRIBUTING file on the .github repo? https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md

Maybe we can just get rid of this file altogether.

Reply:
you are so right, removing